grafana/update_db_config.py

```diff
@@ -29,8 +29,11 @@
     "dryrun": 52,
     "testnet": 105
 }
-do_node_ips = ["143.110.234.143", "138.68.11.38", "143.110.234.143"]
-local_disk_node_ips = ["3.236.64.29", "18.116.33.241", "52.53.250.187", "35.82.29.182", "34.229.131.248", "54.189.61.183", "54.188.66.208", "34.216.159.65", "54.212.179.219", "44.235.31.107", "44.241.141.240", "44.237.240.249", "44.225.100.239", "34.222.190.168", "44.230.211.156"]
+do_node_ips = []
+local_disk_node_ips = ["3.236.64.29", "18.116.33.241", "52.53.250.187", "35.82.29.182", "34.229.131.248",
```

Comment:
can you remove this part since we discussed it wasn't required

Reply:
done